src/main/resources/com/dabsquared/gitlabjenkins/GitLabPushTrigger/config.jelly

```diff
@@ -18,6 +18,9 @@
     <f:entry title="Add note with build status on merge requests" field="addNoteOnMergeRequest">
       <f:checkbox default="true" />
     </f:entry>
+    <f:entry title="Use GitLab CI features (GitLab 8.1 required!)" field="addCiMessage">
```

Comment:
:/ I had one, but needed to create a new branch.. Guess I missed some lines in the progress